web-client/src/presenter/actions/getUserAction.test.js

```diff
@@ -0,0 +1,44 @@
+import { applicationContextForClient as applicationContext } from '../../../../shared/src/business/test/createTestApplicationContext';
+import { getUserAction } from './getUserAction';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+
+describe('getUserAction', () => {
+  const mockUser = {
```

Comment:
Seems like another opportunity to use our shared `mockUsers`

Reply:
This mock is just a few lines long, and the simplicity of this action really doesn't warrant importing mocks from elsewhere in this case, IMO.